lighthouse-extension/bundle-builder.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Script to bundle lighthouse entry points so that they can be run
+ * in the browser (as long as they have access to a debugger protocol Connection).
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const LighthouseRunner = require('../lighthouse-core/runner');
```

Comment:
this is all almost exactly the same as it was in the gulpfile, just unindented a few times. One good piece of news: our `astw` hack is no longer necessary because `insert-module-globals` moved off off `lexical-scope` in https://github.com/browserify/insert-module-globals/pull/76 :)

Reply:
oh, and no objection to future moving to a different bundler or whatever. Left exactly the same to stay out of that question for now :)

It should be even easier since bundling is isolated from the specifics of extension building.